### PR TITLE
Reflection

### PR DESCRIPTION
**Pull request by jannelie**
As a developer, I want to have a reflection paragraph in our final report, so that we know what to during a next project.

### CODE REVIEW

**Comment by ardwennem2** [comment]
Ik vind het erg goed - Approved

**Comment by HarmenKroon** [comment]
Ik vind het voldoende - Approved

**Comment by HarmenKroon** [comment]
Approved! Ik vind het voldoende

File: docs/final report/Reflection.tex [hunk 1]
```diff
@@ -1,4 +1,10 @@
...
...n with the bot and a much better strategy. At the start of this project we expected thought that we had to implement a bot. That we had to write a strategy for our bot so it was able to play in the Tygron game. Unfortunately we were wrong. We also had to implement the connector. We had to make sure all the information we needed for the bot, was implemented in the connector. So instead of using the percept, we had to write the percepts. This caused a big change in our plan. Because instead of focusing on a nice strategy, we needed to focus on implementing code to get percepts before even thinking about a strategy. So our final product is now a basic bot, who is able to do basic steps, like building and buying. But a very nice strategy or nice interaction with other bots, is not really there. The improvement for our bot is to have this nice strategy. We weren't able to have this, because we didn't have the information in the connector. Now we have. So for next years it should be able to build the nice strategy we had in mind. Because they don't have to implement anything in the connector, they should be able to think of a strategy and implement this using our code from the connector.
```
Comment (nskartoredjo) [inline docs/final report/Reflection.tex]: - 'and a much better strategy` => `and to have a much better strategy`
- 'a bot` => what kind of bot? or do you mean only implementing the bot?
- 'information in the connector` => information or implementation?
- 'for next years..` => why has this sentence been added?
- to many 'nice` repeats. Also explain what could be considered as 'nice`

**Comment by jannelie** [comment]
I improved the teksts folowing your feedback.

File: docs/final report/Reflection.tex [hunk 2]
```diff
@@ -1,7 +1,7 @@
...
...plement a bot for the Tygron game and that the connector had all the information we needed. That we had to write a strategy for our bot so it was able to play in the Tygron game. Unfortunately we were wrong. We also had to implement the connector. We had to make sure all the information we needed for the bot, was implemented in the connector. So instead of using the percept, we had to write the percepts. This caused a big change in our plan. Because instead of focusing on a strategy, we needed to focus on implementing code to get percepts before even thinking about a strategy. So our final product is now a basic bot, who is able to do basic steps, like building and buying. But a very nice strategy or interaction with other bots, is not really there. The improvement for our bot is to have this nice strategy. We weren't able to have this, because we didn't have the information implemented in the connector. Now we have. So if we had another ten weeks for this project we should be able to build the strategy we had in mind. Because then we don't have to implement anything new in the connector, we should be able to think of a strategy and implement this using our code from the connector.
```
Comment (HarmenKroon) [inline docs/final report/Reflection.tex]: 'expected thought' gebruik niet twee werkwoorden
De laatste zin omdraaien. Begin geen zin met because

**Comment by jannelie** [comment]
I changed the text according to your feedback

**Comment by HarmenKroon** [comment]
Approved

**Comment by nskartoredjo** [comment]
Approved

**Pull request merged**